.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.234.0/containers/go
+{
+	"name": "Go",
+	"build": {
+		"dockerfile": "Dockerfile"
+	},
+	"runArgs": [
+		"--cap-add=SYS_PTRACE",
+		"--security-opt",
+		"seccomp=unconfined"
+	],
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"go.toolsManagement.checkForUpdates": "local",
+		"go.useLanguageServer": true,
+		"go.gopath": "/go",
+		"go.formatTool": "goimports",
+		"editor.formatOnSave": true
+	},
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"golang.Go",
+		"mhutchie.git-graph",
+		"ms-vscode.makefile-tools"
+	],
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	"forwardPorts": [
+		3000
+	],
```

Comment:
I think we should forward a different port than `3000`, in case there are any full stackers who want to run both apps together...Local system can adjust automatically, obviously, but figure this approach would be a little "cleaner"...

Reply:
That's a great point! It's very likely that folks will run both services together. I'll do this now in another PR so that https://github.com/omshub/core-api/pull/5 and this can be easily rebased.

Reply:
Sounds good...in that case, approved!

Reply:
Here you go! https://github.com/omshub/core-api/pull/10

Reply:
Also, regarding the new port number, not sure if there are any canonical/conventional server-side ports in the Go space (i.e., as `3000` is in the React world), but if not, we could do a [meme number](https://en.wikipedia.org/wiki/George_P._Burdell) like `1928` for GT

Reply:
I went with plain 8080, but `1927` is 👌